chore(blog): announcing AsyncAPI & Spectral collaboration (#663)
asyncapi/website: pages/blog/creating-consistency-announcing-asyncapi-spectral-together.md

Consistency is a top concern for companies across the globe.

At Stoplight, we’re seeking to help your organization achieve consistency. We believe that API descriptions should be
treated the same way as code. APIs should have style guides with rules, and be reviewed to ensure they are descriptive,
readable, and concise for the developers who use them. Developer experience is, after all, another top concern for
companies.

So, we created our open-source JSON linting tool Spectral. It promotes standards with the flexibility for custom rule
creation to validate and lint any JSON.

For example, while an AsyncAPI or OpenAPI document might be entirely valid, it could be missing important fields like
descriptions for parameters or have other problematic design issues. Spectral can warn about possible improvements to
ensure API definitions can reach their full potential, without needing to have special code owners review every change
to the documents.

Although Spectral is mostly referenced in the context of OpenAPI, AsyncAPI has been an integral

> part of Spectral for quite some time now. In fact, later this month our Spectral AsyncAPI ruleset will celebrate its

Author: ```suggestion
part of Spectral for quite some time now. In fact, last month our Spectral AsyncAPI ruleset celebrated its
```
